### PR TITLE
Feature/cax portal@cplp 911 my user account page

### PR DESCRIPTION
**Pull request by floriansommerraysono**
My User Account Page & User Account Page with mock data + smaller component adjustments + small fixes
- CPLP-909
- CPLP-910
- CPLP-911

### CODE REVIEW

**Comment by oyo** [comment]
I guess that code duplication would be resolved once the API is connected. Otherwise looks okay.

**Comment by floriansommerraysono** [comment]
Hi, I check the unit tests and somehow, I think by a merge, an old line of code was re-added to the component export index file. I checked it again and this file is not used anymore and was deleted. I committed a fix and unit tests are now running. Greetings

**Review by oyo — APPROVED** [review]
looks good from my side now - we can ignore the Sonar code duplication findings for now.

**Comment by floriansommerraysono** [comment]
@oyo I just checked those duplicated lines from SonarCloud and those are mostly mock data. They will be removed anyway. Thanks for the approval.

File: cx-portal-shared-components/src/theme.ts [hunk 1]
```diff
@@ -476,6 +476,14 @@ export const theme = createTheme({
           padding: '16px 32px',
           borderColor: palette.border.border01,
         },
+        columnHeaderCheckbox: {
```
Comment (PI-FerasAhmad) [inline cx-portal-shared-components/src/theme.ts:479]: where is this style used? will be replaced with the checkbox style from cx-portal-shared-components/src/components/basic/Table/index.tsx ?

Comment (floriansommerraysono) [reply cx-portal-shared-components/src/theme.ts:479]: Yes those are the CSS class overwrites for the MuiDataGrid and are the style adjustments you made. I just moved them from the component to the global styleOverrides.

File: cx-portal-shared-components/src/components/content/UserDetails/UserDetailCard.tsx [hunk 2]
```diff
@@ -1,23 +1,51 @@
 import EditOutlinedIcon from '@mui/icons-material/EditOutlined'
 import { Box, List, ListItem } from '@mui/material'
-import { IconButton } from '../IconButton'
-import { Typography } from '../Typography'
+import { IconButton } from '../../basic/IconButton'
+import { Typography } from '../../basic/Typography'
+import { Chip } from '../../basic/Chip'
 
-export interface UserCardProps {
+interface UserCardProps {
```
[the remark this answers is not in the document]
Comment (floriansommerraysono) [reply cx-portal-shared-components/src/components/content/UserDetails/UserDetailCard.tsx:7]: Good find! Yes this is a duplication. Going to improve that.

**Review by PI-FerasAhmad — APPROVED** [review]

**Pull request merged**